src/maintainer/knowledge_base.rst

```diff
@@ -1148,23 +1148,7 @@ because these symbols are in fact available. To do so, add
 PyPy builds
 ===========
 
-To use the PyPy 3.6 or 3.7 builds you can do the following,
-
-.. code-block:: bash
-
-   conda create -n pypy36  pypy python=3.6
-   conda create -n pypy37  pypy python=3.7
```

Comment:
Can we keep an instruction like this to switch pypy version?

Reply:
Do you mean "keep an instruction in the knowledge_base document" or "keep an instruction with the `pypy python=x.y` syntax"?

Reply:
It seems the `pypy python=x.y` syntax is preferable. Using `conda create -n pypy3.7 pypy3.7` is missing these packages:
```diff
>   pip                conda-forge/noarch::pip-22.0.4-pyhd8ed1ab_0
>   pypy               conda-forge/noarch::pypy-7.3.7-0_pypy37
>   python             conda-forge/linux-64::python-3.7.12-0_73_pypy
>   python_abi         conda-forge/linux-64::python_abi-3.7-2_pypy37_pp73
>   setuptools         conda-forge/linux-64::setuptools-60.10.0-py37h9c2f6ca_0
>   wheel              conda-forge/noarch::wheel-0.37.1-pyhd8ed1ab_0
```